data/global listeners.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2024 by RisingLeaf(https://github.com/RisingLeaf)
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Unlock Cloaking Key"
+	invisible
+	landing
+	to offer
+		and
+			not "global: found cloaking"
+			or
+				has "flagship attribute: cloak"
+				has "outfit (all): Cloaking Device"
```

Comment:
What if the player gets an escort with builtin cloaking?

Reply:
How would I check for that?

Reply:
You'll probably need to implement a new condition for this, something like "flagship attribute", but for the whole fleet.

Reply:
see #10142